concourse/tasks/gcloud-get-secret.yaml

```diff
@@ -0,0 +1,30 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  secret-version: "latest"
+  secret-name:
+
+inputs:
+- name: credentials
+
+outputs:
+- name: gcp-secret-manager
+
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+  PROJECT: "gcp-guest"
+
+run:
+  path: sh
+  args:
+  - -exc
+  - |
+    gcloud auth activate-service-account --key-file=$GOOGLE_APPLICATION_CREDENTIALS
+    gcloud secrets versions access ((secret-version)) --secret=((secret-name)) --project=$PROJECT > gcp-secret-manager/((secret-name))
```

Comment:
params are actually environment variables, so `((secret-version))` should be `$secret-version` (as currently named)

Reply:
Done